bundle_notification.gemspec

```diff
@@ -28,15 +29,15 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'actionmailer', '>= 4.0', '< 5.3'
-  spec.add_development_dependency 'bundler', '~> 1.14'
+  spec.add_development_dependency 'actionmailer', '~> 6.0.0.beta1'
+  spec.add_development_dependency 'bundler', '~> 2.2.22'
   spec.add_development_dependency 'factory_girl'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'simplecov'
   spec.add_development_dependency 'sqlite3'
 
-  spec.add_dependency 'activerecord', '>= 4.0', '< 5.3'
-  spec.add_dependency 'activesupport', '>= 4.0', '< 5.3'
+  spec.add_dependency 'activerecord', '6.0.0.beta1'
+  spec.add_dependency 'activesupport', '6.0.0.beta1'
```

Comment:
Should we keep support for lower versions? And why pin it to the `6.0.0-beta1` version? We are at Rails 6.1.4 already.

Reply:
I don't think we can support lower version here. Because there's a conflict in BO where `activesupport 6.0.0.beta1 ` is needed, whereas here it's bounded to `5.3`

Reply:
> I don't think we can support lower version here.

Why?

> Because there's a conflict in BO where `activesupport 6.0.0.beta1` is needed, whereas here it's bounded to 5.3

This is a gem, though, and not meant to be usable only by the Backoffice. You can specify the version with `~> 6.1`, if you want to allow all versions >= 6 and < 7.